utils/log4cplus/log4cplus_d.conf

```diff
@@ -0,0 +1,10 @@
+log4cplus.rootLogger=TRACE,STDOUT,ALL_MSGS
```

Comment:
What's the difference between TRACE and INFO?

Reply:
They represent different levels of log output. Priority from low to high includes TRACE ->DEBUG ->INFO ->WARN